srcpkgs/vala/template

```diff
@@ -15,13 +16,6 @@ distfiles="${GNOME_SITE}/vala/${version%.*}/vala-${version}.tar.xz"
 checksum=67d9bb4656d8fe04bcfc1ed7ff35d191df425923de46c921ae3c3d996eee8040
 shlib_provides="libvalaccodegen.so"
 
-# for valadoc
-if [ "$CROSS_BUILD" ]; then
-	hostmakedepends+=" vala"
-else
-	make_build_args="VALAC="${XBPS_BUILDDIR}/${pkgname}-${version}/compiler/valac""
-fi
-
 pre_configure() {
 	autoreconf -if
 }
```

Comment:
No need for this and the automake/libtool dep now that we don't patch this package anymore